compose/material/material/src/skikoMain/kotlin/androidx/compose/material/AtomicActual.kt

```diff
@@ -14,17 +14,16 @@
  * limitations under the License.
  */
 
-package androidx.compose.foundation
+package androidx.compose.material
 
-actual class AtomicLong actual constructor(value: Long) {
```

Comment:
Let's add `## Breaking change` to the PR description. It will be added manually to the changelog later (or via some future script)

Reply:
> actual was a (public) class (not a typealias) and it was using the declaration

We should write it from the user perspective. The user sees only available classes, not that there are aliases/actuals. We should enumarate all removed classes and sourceSets (jsMain, iosMain, wasmMain).